azure-kusto-ingest/azure/kusto/ingest/kusto_ingest_client.py

```diff
@@ -11,8 +11,8 @@
 from azure.storage.common import CloudStorageAccount
 from azure.kusto.data import KustoClient
```

Comment:
Missing a blank line to separate third-party imports from local imports.

Reply:
I am just adding tests to 'KustoIngestClient' that will also test the 'ResourceManager',
that will be reviewed in another pull request.